editor/editor_build_profile.cpp

```diff
@@ -79,38 +104,215 @@ const bool EditorBuildProfile::build_option_disabled_by_default[BUILD_OPTION_MAX
 const bool EditorBuildProfile::build_option_disable_values[BUILD_OPTION_MAX] = {
 	// This maps to SCons build options.
 	true, // 3D
-	true, // PHYSICS_2D
-	true, // PHYSICS_3D
-	true, // NAVIGATION
+	true, // NAVIGATION 2D
+	true, // NAVIGATION 3D
 	true, // XR
+	false, // OPENXR
+	false, // WAYLAND
+	false, // X11
 	false, // RENDERING_DEVICE
-	false, // OPENGL
+	false, // FORWARD_RENDERER
+	false, // MOBILE_RENDERER
 	false, // VULKAN
+	false, // D3D12
+	false, // METAL
+	false, // OPENGL
+	true, // PHYSICS_2D
+	false, // PHYSICS_GODOT_2D
+	true, // PHYSICS_3D
+	false, // PHYSICS_GODOT_3D
+	false, // PHYSICS_JOLT
 	false, // TEXT_SERVER_FALLBACK
-	false, // TEXT_SERVER_COMPLEX
+	false, // BUILD_OPTION_TEXT_SERVER_ADVANCED
```

Comment:
`BUILD_OPTION_` is implicit in these comments. (Same for other occurrences.)
```suggestion
	false, // TEXT_SERVER_ADVANCED
```

Reply:
You missed this one.